testing/php7-yaml/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+
+pkgname=php7-yaml
+_pkgreal=yaml
+pkgver=2.0.0
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP${_phpver}"
+url="http://bd808.com/pecl-file_formats-yaml/"
+arch="all"
+license="MIT"
+makedepends="php${_phpver}-dev autoconf yaml-dev"
```

Comment:
Even when "some$mangling-could" happen?
Bash would not confuse a variable $mangling as $mangling-could. Even though bash variable names would not consider dash (i.e. redirect operator, or minus mathematical operator) as part of the name.

Assuming `_varname=7`, I imagine then the convention is OK with `"foo$_varname-bar"` (because of dash) but I would have to `${...}` if I had no dash after the variable name (e.g. `"foo${_varname}bar"`).

Is that the convention guideline? I haven't found notes about that in http://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package